### ISSUE TITLE
mysql build fails

### ISSUE THREAD

**Issue by bkauler** [sabotage-linux/sabotage#419]
Hi,
I went through all the steps in the readme, from cloning the repo, right up to running "butch install world", for x86_64 native build.

It completed, except for two build fails, php and mysql.

Here is the log for mysql:

**Comment by bkauler** [comment]
[build_mysql.log.txt](https://github.com/sabotage-linux/sabotage/files/244401/build_mysql.log.txt)

**Comment by rofl0r** [comment]
it's missing -lterminfo in LDFLAGS. could you check if adding that fixes it?

**Comment by bkauler** [comment]
Yes, it compiled!

I edited pkg/mysql, inserted this line:

```
  -DDEFAULT_CHARSET_HOME="/" \
  -DCMAKE_EXE_LINKER_FLAGS="-lterminfo $optldflags" \
      ..

```

I didn't know if I should put in the $optldflags, did anyway.

**Comment by rofl0r** [comment]
thanks, fixed via  	5543749
